bookmarks/tests/test_metadata_view.py

```diff
@@ -26,8 +108,90 @@ def test_manifest_respects_context_path(self):
         response_body = response.json()
         expected_body = {
             "short_name": "linkding",
+            "name": "linkding",
```

Comment:
Nit: Instead of duplicating everything, maybe extract the common parts and reuse them in individual tests.

Reply:
Just noticed that a lot of URLs are different. Nevermind then, can be improved if more tests are added. As it is it's a pretty decent snapshot test.